src/admin-views/widgets/components/fields/number.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
This is funny because ET+ has it's own "fields api" that has this and other support added to the number field. Something to think about in the future there for us to revisit

Reply:
We def need to have a good convo around Fields, lots of stuff that might be good to come from Common in a standard component.